libs/error-analysis/src/lib/ErrorAnalysisDashboard/ErrorAnalysisDashboard.tsx

```diff
@@ -624,10 +624,6 @@ export class ErrorAnalysisDashboard extends React.PureComponent<
                     <GlobalExplanationTab
                       jointDataset={this.state.jointDataset}
                       metadata={this.state.modelMetadata}
-                      globalImportance={this.state.globalImportance}
```

Comment:
can this.state.globalImportance just be removed now?  I'm a bit confused how GlobalExplanationTab gets this info now.